app/src/componentLibrary/autocompletes/singleAutocomplete/singleAutocomplete.jsx

```diff
@@ -116,15 +137,23 @@ export class SingleAutocomplete extends Component {
                           !value && openMenu();
                           onFocus();
                         },
-                        onBlur: () => {
-                          onBlur();
+                        onBlur: (e) => {
+                          if (createWithoutConfirmation) {
+                            selectItem(inputValue);
+                          }
+                          onBlur(e);
+                          isOptionUnique &&
+                            isOptionUnique(
+                              inputValue ? !options.find((v) => v === inputValue) : null,
+                            );
```

Comment:
Why we need to do it here?
And may be better to rename it to `setIsOptionUnique` as it is an action, not just boolean.

Reply:
I have no possibility to compare options values and value somewhere outside